src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -191,20 +193,47 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
       }
     }
 
-    if (this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers) {
+    const isSimulcastPolicy = this.isSimulcastUplinkPolicy(
+      this._configuration.videoUplinkBandwidthPolicy
+    );
+    let enableSimulcast =
+      this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers || isSimulcastPolicy;
+
+    if (enableSimulcast) {
       if (!this._configuration.enableUnifiedPlanForChromiumBasedBrowsers) {
-        this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = false;
         this.logger.info(
           'Simulcast requires enabling WebRTC Unified Plan for Chromium-based browsers'
         );
-      } else if (browserBehavior.hasChromiumWebRTC()) {
+        enableSimulcast = false;
+      } else if (this._configuration.videoUplinkBandwidthPolicy && !isSimulcastPolicy) {
```

Comment:
isn't the `this._configuration.videoUplinkBandwidthPolicy` check redundant with the body of `isSimulcastUplinkPolicy`?

Reply:
isSimulcastPolicy is only set if there is a policy in configuration and it is a simulcast policy. This is the case when they pass in a non-simulcast policy but turn on Simulcast via `enableSimulcastForUnifiedPlanChromiumBasedBrowsers` flag.

Reply:
Are you sure that's what that means?

As a builder, I might want to enable simulcast _support_  in the SDK without turning on simulcast for a particular meeting, and if I specify a policy or a flag, I don't expect the SDK to override it.

I see the hierarchy as:

* If they pass a policy, use that policy. If the policy conflicts with browser support, either continue to work (simulcast degrades into unicast), or throw an error.
* If they don't pass a policy, use a flag like `enableSimulcast` to decide what to do. If their preference (`enableSimulcast`) conflicts with browser support, either continue to work (simulcast degrades into unicast), or throw an error.
* If they don't pass a policy or specify a preference, do the default thing, which might vary between browsers based on support.

Reply:
I do not override the policy in this case right. This case is when they pass in a non-simulcast policy but has that simulcast flag on then we will clear out the simulcast flag.

Reply:
Update the code to throw error when passing invalid values.